Assets/MixedRealityToolkit/StandardAssets/Shaders/MixedRealityStandard.shader

```diff
@@ -679,6 +678,13 @@ Shader "Mixed Reality Toolkit/Standard"
 #else
                 o.scale.z = length(mul(unity_ObjectToWorld, float4(0.0, 0.0, 1.0, 0.0)));
 #endif
+                // uv3.y will contain a negative value when rendered by a UGUI and ScaleMeshEffect.
+                if (v.uv3.y < 0.0)
+                {
+                    o.scale.x *= v.uv2.x;
+                    o.scale.y *= v.uv2.y;
+                    o.scale.z *= v.uv3.x;
```

Comment:
>3.x [](start = 37, length = 3)

Did you mean for this to be uv3?  And X?  Not sure if this is a copy/paste error or intentional because Z is a special case for UGUI.

If it's intentional, add a comment explaining why.